### PR TITLE
Reduce vtctld webdriver test flakiness.

### PR DESCRIPTION
**Pull request by thompsonja**
BUG=34072167

### CODE REVIEW

File: test/vtctld_web_test.py [hunk 1]
```diff
@@ -94,7 +94,11 @@ def _get_keyspaces(self):
 
   def _get_keyspace_element(self, keyspace_name):
     """Get a specific keyspace element given a keyspace name."""
-    return self.driver.find_element_by_id('%s-card' % keyspace_name)
+    element_id = '%s-card' % keyspace_name
+    wait = WebDriverWait(self.driver, 5)
```
Comment (michael-berlin) [inline test/vtctld_web_test.py]: Can you please move the 5 into a constant? This way the code is also going to be more readable.

Comment (thompsonja) [inline test/vtctld_web_test.py]: Done.

**Comment by thompsonja** [comment]
We do not have any documentation on writing webdriver tests, though the wait "trick" is a fairly common practice that is probably gone in other test frameworks like protractor.

**Comment by michael-berlin** [comment]
SG :)

**Pull request merged**